### ISSUE TITLE
Can not disable output to console on accepted connection

### ISSUE THREAD

**Issue by olingerc** [miguelgrinberg/Flask-SocketIO#435]
Every time a client connects to my server (debug true or false) my console prints: 
```(8369) accepted ('10.110.156.234', 37414)```
I checked with werkzeug and flask but I fail to disable this message. Would you know where it is coming from?

**Comment by miguelgrinberg** [comment]
That's coming from the eventlet wsgi server: https://github.com/eventlet/eventlet/blob/master/eventlet/wsgi.py#L882

**Comment by olingerc** [comment]
Going through your function that actually starts the wsgi server, I see that you forward the log_output paramter given when running socketio.run()
```
elif self.server.eio.async_mode == 'eventlet':
            def run_server():
                import eventlet
                import eventlet.wsgi
                eventlet_socket = eventlet.listen((host, port))

                # If provided an SSL argument, use an SSL socket
                ssl_args = ['keyfile', 'certfile', 'server_side', 'cert_reqs',
                            'ssl_version', 'ca_certs',
                            'do_handshake_on_connect', 'suppress_ragged_eofs',
                            'ciphers']
                ssl_params = {k: kwargs[k] for k in kwargs if k in ssl_args}
                if len(ssl_params) > 0:
                    for k in ssl_params:
                        kwargs.pop(k)
                    ssl_params['server_side'] = True  # Listening requires true
                    eventlet_socket = eventlet.wrap_ssl(eventlet_socket,
                                                        **ssl_params)

                eventlet.wsgi.server(eventlet_socket, app,
                                     log_output=log_output, **kwargs)
```
However I still get the output?

Going through the wsgi source code, I think that this specific output can not be disabled via the log_output parameter?

Anyway, I think the problem is not on your side.

**Comment by miguelgrinberg** [comment]
Did you try setting a high log level on the logging object? See https://docs.python.org/2/library/logging.html#logging-levels.

If you set it to `ERROR` or `WARNING` the regular server output should be suppressed, I think.

**Comment by olingerc** [comment]
Yes, I've put `logging.basicConfig(level=logging.ERROR)` at the top of my main module plus `ERROR` on all my custom loggers without success.

**Comment by miguelgrinberg** [comment]
You can pass your own logger object to eventlet's wsgi server in the `log` argument:

    my_logger = logging.getLogger('my-logger')
    my_logger.setLevel(logging.ERROR)
    socketio.run(app, log=my_logger)

**Comment by olingerc** [comment]
FYI:
This was a bug in eventlet which was resolved with version [0.21.0](http://eventlet.net/doc/changelog.html). Starting our app with `socketio.run(app, log_output=False)` is now finally less chatty.
